modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogValidationException.java

```diff
@@ -26,6 +27,15 @@
 public class CatalogValidationException extends IgniteInternalException {
     private static final long serialVersionUID = 2692301541251354006L;
 
+    /**
+     * Constructor.
+     *
+     * @param message Error message.
+     */
+    public CatalogValidationException(String message) {
+        super(Common.INTERNAL_ERR, message);
```

Comment:
It seems worth adding a comment why we don't care about the error code.

p.s. I didn't find where these exceptions are now mapped to STMT_VALIDATION_ERR code.
Maybe I didn't look well enough or it will be added in [1].

[1] https://issues.apache.org/jira/browse/IGNITE-20149

Reply:
decided to introduce new code for this